### DISCUSSION TITLE
Input file reading error

### DISCUSSION THREAD

**Discussion by jk262626** [nakib/elphbolt#140] [General]
Hi Nakib,

I sometimes have a problem with reading input file in electron setup step. 
This happens randomly, like sometimes it works well and sometimes it doesn't work with the exactly same input files and settings. 

I'm attaching the slurm file.
[slurm-30735134.zip](https://github.com/nakib/elphbolt/files/15406979/slurm-30735134.zip)

It's saying like there is an error in opening input.nml file during the eletron.f90. I'm curious why it doesn't work during a specific process. For example, opening input.nml worked well in the process of Wannier.f90 something like that. Would there be a solution for this problem?

**Comment by nakib** [comment]
Hi jk262626,

I have not encountered this error on the 8 or so machines that I have run this code on. It looks like something related to how your cluster is set up. Maybe your cluster maintainer can help with this as this has nothing to do with the code.

By the way, I see that you are using a pretty old version of the code. I suggest that you update to the latest develop branch.

Best,
Nakib

**Comment by jk262626** [comment]
Yes, it was the problem of the cluster system I was using. I have been doing calculation on lustre file system and it had some problems. I moved the working directory to /home, and the problem has been solved!

And I updated Elphbolt to the latest version.
Thank you Nakib!
